L1Trigger/L1TGlobal/test/testVectorCode_data.py

```diff
@@ -192,7 +192,8 @@
 
 process.load("L1Trigger.GlobalTriggerAnalyzer.l1GtTrigReport_cfi")
 process.l1GtTrigReport.L1GtRecordInputTag = "simGtStage2Digis"
-process.l1GtTrigReport.PrintVerbosity = 2
+process.l1GtTrigReport.PrintVerbosity = 0 #EF DEBUG
+#process.l1GtTrigReport.PrintVerbosity = 2 #EF DEBUG
```

Comment:
Would it be possible to remove the commented line? It's basically the same thing as the line above.